my-school/src/index.css

```diff
@@ -1,3 +1,5 @@
+@import url('https://fonts.googleapis.com/css2?family=Nunito:ital,wght@0,300;0,400;0,700;1,300;1,400;1,700&display=swap');
```

Comment:
In the end we may want to download and include the font locally - we'll see what kind of latency we get on font loading. Just a note